tests/testthat/test-verify.R

```diff
@@ -0,0 +1,41 @@
+testthat::test_that("verify returns the same object for empty teal_data", {
+  tdata1 <- teal_data()
+  testthat::expect_identical(tdata1, verify(tdata1))
+  testthat::expect_identical(tdata1@verified, TRUE)
+})
+
+testthat::test_that("verify returns the same object for teal_data where code was run in within/eval_code", {
+  tdata1 <- teal_data()
+  tdata1 <- within(tdata1, {
+    a <- 1
+    b <- a^5
+    c <- list(x = 2)
+  })
+  testthat::expect_identical(verify(tdata1)@verified, TRUE)
+  testthat::expect_identical(verify(tdata1)@code, tdata1@code)
+  testthat::expect_identical(verify(tdata1)@env, tdata1@env)
+})
```

Comment:
Both of these test actually test the same thing: that `verify` returns `x` when `@valid` is `TRUE`. Please collate.

Reply:
totally - changed in 1d23cb7d9bdd24559f2f9d918b97a14684b47f1d